QRCoder/QRCoder.csproj

```diff
@@ -33,4 +35,8 @@
     <PackageReference Include="System.Drawing.Common" Version="4.5.1" />
   </ItemGroup>
 
+  <PropertyGroup>
```

Comment:
Workaround for ```dotnet pack``` multi-targeting net35. https://github.com/Microsoft/msbuild/issues/1333#issuecomment-296346352